Standards/scs-XXXX-vN-taxonomy-of-failsafe-levels.md

```diff
@@ -13,11 +13,127 @@ There does exist very detailed lists of risks and what consequences there are fo
 
+<!--
+TODO: goals vs. non-goals
+TODO: Resolve confusion between high availability vs. disaster recovery vs. redundancy vs. backups
+TODO: What time frame do we look at? (so called Recovery Time Objecte aka RTO)
+TODO: how does this relate to Business Continuity Planning (BCP)
+-->
+
+## Context
+
+Some standards provided by the SCS project will talk about or require procedures to back up resources or have redundancy for resources.
+This decision record should discuss, which failure threats are CSP-facing and will classify them into several levels.
+In consequence these levels should be used in standards concerning redundancy or failure safety.
+
+Based on our research, no similar standardized classification scheme seems to exist currently.
+Something close but also very detailed is the [BSI-Standard 200-3 (german)][bsi-200-3] published by the German Federal Office for Information Security.
+As we want to focus on IaaS and K8s resources and also have an easily understandable structure that can be applied in standards covering replication, redundancy and backups, this document is too detailed.
+
+[bsi-200-3]: https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Grundschutz/BSI_Standards/standard_200_3.pdf?__blob=publicationFile&v=2
+
+## Decision
+
+### Failsafe Levels
+
+This Decision Record defines **four** failsafe levels, each of which describe what kind of failures have to
+be tolerated by a provided service.
+
+In general, the lowest, **level 1**, describes isolated/local failures which can occur very frequently, whereas
+the highest, **level 4**, describes relatively unlikely failures that impact a whole or even multiple datacenter(s):
+
+| Level | Probability | Impact      | Examples |
+| -     | -           | -           | -       |
+| 1     | Very High   | Local       | Disk failure, RAM failure, software bug |
+| 2     | High        | Moderate    | Rack outage, power outage, small fire |
+| 3     | Medium      | High        | Regional power outage, huge fire, orchestrated cyber attack |
+| 4     | Low         | Very high   | Natural disaster |
+
+<!--
+TODO: find better categories for impact (maybe site-wide, regional, and so on?) also: scale vs. impact?
+-->
+
+For example, a provided service with failsafe level 2 tolerates a rack outage (because there is some kind of
+redundancy in place.)
+
+From a cloud service provider (CSP) perspective, supporting these failure levels has the following *general*
+consequences:
+
+* **Level 1**: CSPs MUST operate replicas for important components (e.g., RAID, replicated volume backend, uninterruptible power supply).
+* **Level 2**: CSPs SHOULD operate hardware in dedicated availability zones (AZs).
+* **Level 3**: CSPs SHOULD operate hardware in dedicated regions.
+* **Level 4**: Depending on the regions, CSPs may not be able to save user data from such catastrophes.
+
+More specific guidance on what these levels mean on the IaaS and KaaS layers will be provided in the sections
```

Comment:
I integrated a table to show both consequences for CSPs and users. We may discuss again, because you mentioned to already user AZs in Level 2, while this is imho more a level 3 consequence.